### ISSUE TITLE
Can I do bonjour service discovery with this ?

### ISSUE THREAD

**Issue by sander76** [al45tair/asyncdns#2]
I am using python zeroconf for getting the ip address of a local device with a specific mdns/bonjour name.

Something comparable to:
https://github.com/jstasiak/python-zeroconf#how-do-i-use-it

Is this library suited for doing this too ?

**Comment by al45tair** [comment]
Yes, you can use asyncdns to look up services using the `MulticastResolver`. Try using the pymdns script in the "scripts" folder as an example, e.g.:

    ./scripts/pymdns _http._tcp.local

**Comment by sander76** [comment]
@al45tair That's great. Going to try it ! Thanks.

**Comment by sander76** [comment]
@al45tair Sorry to bother you again. I have little experience doing this.
Your suggestion above works. However, I have multiple devices carrying the same bonjour name. If I run the script I receive only one result. If I re-run I get the next result, and so on. Is it possible to gather all results in one go ? I should I just put the script in a loop or something ?
